package.json

```diff
@@ -88,7 +96,8 @@
         "start": "npm run develop",
         "start-80": "sudo gatsby develop -p 80",
         "serve": "gatsby serve",
-        "test": "npm run eslint && npm run stylelint",
+        "lint": "npm run eslint && npm run stylelint",
+        "test": "jest",
```

Comment:
If you update this here, you need to update all the `.yml` scripts that uses the `test` command.

Reply:
Noted @sean-binary. I'll be sure not to break any of our flow. This PR is still at draft so expect some changes 🙏🏼